src/utils/language.ts

```diff
@@ -5,19 +5,20 @@ import { systemLanguageRef } from './refs';
 
 // Language: Language Code in books/translations
 export const languageCodeMap: Record<string, string> = {
-  English: 'en',
-  French: 'fr',
-  German: 'de',
-  Portuguese: 'pt',
   Arabic: 'ar',
   Catalan: 'ca-ES',
-  Spanish: 'es',
+  Chinese: 'zh-CN',
   Dutch: 'nl',
+  English: 'en',
+  French: 'fr',
+  German: 'de',
   Gujarati: 'gu',
-  Turkish: 'tr',
   Korean: 'ko',
-  Swedish: 'sv',
-  'Simplified Chinese': 'zh-CN',
```

Comment:
Why rename `Simplified Chinese` to  `Chinese` ?

Reply:
oh sorry my mistake